### DISCUSSION TITLE
How to update value in a menu item?

### DISCUSSION THREAD

**Discussion by dj-fiorex** [forntoh/LcdMenu#9] [Q&A] [answered]
Hi, this is my second issue, because i really love your library! 
Can i show a menu item that update itself? Like when a pin change or when a variable change/
Thanks

**Comment by forntoh** [comment, marked as answer]
You can update the value of an `ItemInput` in the `loop` function by calling
`menu.getItemAt(``position``)->value = "your new value here"`

Examples:
1. `menu.getItemAt(``your position``)->value = "your new value here"`
1. `menu[``your position``]->value = "your new value here"`
1. `menu[``your position``][``your sub position``].value = "your new value here"`

`1.` and `2.` are the same
`3.` will change the value of the item at ` ``your sub position`` ` in the submenu at ` ``your position`` `

You can also use `[menu.getCursorPosition()]` in place of `menu.[``your position``]` if your cursor is on the desired menu

**Comment by B1ackt34** [comment]
Great library !!!
Does it work with ITEM_STRING_LIST too?
In my project i have:
```
String fanspeeds[] = {"0%", "25%", "50%", "75%", "100%", "AUTO"};
---
ITEM_STRING_LIST("Fans RPM", fanspeeds, 6, fanspeedsCallback),
```
At some point of my sketch i want to set fanspeeds at "75%" for example ... 
I just call
`menu[``Fans RPM``]->value = "75%" `
or do i need to pass the entire String?
